internal/e2e/node/data/missing-data.json

```diff
@@ -0,0 +1,3 @@
+{
+  "value": 42
```

Comment:
put a distinct value in here so there's never a question that a test passed by accident by reading the value `42` from the wrong file

Reply:
Fixed